src/llmcompressor/modifiers/obcq/base.py

```diff
@@ -6,7 +6,7 @@
 from torch.nn import Module
 from tqdm import tqdm
 
-from llmcompressor.core.state import State
+from llmcompressor.core import State
```

Comment:
Could you check if the pathway that uses always avoids circular import?
Usually then the library import is less strict circular import can be introduced

Reply:
The tests pass and cover importing all modifiers, so I think